Apps/Medication/src/Properties/launchSettings.json

```diff
@@ -4,25 +4,17 @@
     "windowsAuthentication": false,
     "anonymousAuthentication": true,
     "iisExpress": {
-      "applicationUrl": "http://localhost:3003"
+      "applicationUrl": "https://localhost:3003"
     }
   },
   "profiles": {
-    "IIS Express": {
-      "commandName": "IISExpress",
-      "launchBrowser": true,
-      "launchUrl": "swagger",
-      "environmentVariables": {
-        "ASPNETCORE_ENVIRONMENT": "Development"
-      }
-    },
-    "Medication": {
+    "HealthGateway.Medication": {
       "commandName": "Project",
       "launchBrowser": true,
       "environmentVariables": {
         "ASPNETCORE_ENVIRONMENT": "Development"
       },
-      "applicationUrl": "http://localhost:3003"
+      "applicationUrl": "https://localhost:3003;"
```

Comment:
lets not user https since locally we dont have a valid certificate. throughout

Reply:
Reverting to Http